code/ai/aicode.cpp

```diff
@@ -5200,6 +5212,118 @@ void ai_evade()
 float	G_collision_time;
 vec3d	G_predicted_pos, G_fire_pos;
 
+//helper function for create_subject_target_combos
+void set_unset_primary_linking(ship *shp, bool set)
```

Comment:
This can be replaced by
`shp->flags.set(Ship::Ship_Flags::Force_primary_unlinking, set);`
in all the places where it is used.